### ISSUE TITLE
Site design currently broken

### ISSUE THREAD

**Issue by Andre601** [botblock/botblock.github.io#4]
Due to the changes made to the main BotBlock Website is the `docs.botblock.org` website completely broken in its design.

We should find a way to repair it and make it the same look as the BotBlock website itself.

The question now is, how can we do this the best way?
One way could perhaps be to make it like the [Best Practices](https://botblock.org/lists/best-practices) by having a `.vue` file for it in static-site and pull from the `index.md` here to generate the page.
Downside could be, that we no longer have the page under the `docs` subdomain, unless we use that instead or implement redirects for it.
Also, this would open the question if it is even worth keeping the separate docs branch, as the only other page existing would be the one for [JavaBotBlockAPI](https://github.com/botblock/JavaBotBlockAPI) which itself only uses it for hosting the Java code documentation (Javadocs), which I think could be supported in the same way as the Best Practices page?

Either way would it be good to find a solution here to get this resolved in some reasonable way.

**Comment by MattIPv4** [comment]
I am very much leaning towards the view that we simply don't need `docs.botblock.org`, and that we can incorporate some small changes into `botblock.org/docs/libraries` to make sure there is a link out to the Javadocs for JavaBotBlockAPI?

**Comment by Andre601** [comment]
Yeah. Only issue (at least in my case) is that my project has multiple docs, due to it being split into modules, so instead of just having like `https://botblock.org/docs/libraries/javabotblockapi` (Example. Not sure how domains would be handled to link to docs and stuff) would it here probably be more like `https://botblock.org/docs/libraries/javabotblockapi/core` and similar...

But I mentioned this on the Discord that I'm currently leaning towards turning my mess of a code into a single-module project to get rid of a lot of limitations, issues and similar, so that issue above can imo be safely ignored.

**Comment by MattIPv4** [comment]
I'm imagining that those can stay under `docs.botblock.org`, I'm only thinking about removing the homepage that is currently in this repo

**Comment by Andre601** [comment]
Yeah, that would be fine by me.

**Comment by Andre601** [comment]
Question now: Would we just like archive this repo once it's no longer needed, or would it be removed? I personally don't see any real value in having this repo staying here...

**Comment by MattIPv4** [comment]
The repo needs to stay so that the `CNAME` remains active, but we can replace the other contents here with a single `index.html` that does a client-side redirect to `botblock.org/docs/libraries`.
